content/docs/install.md

```diff
@@ -1,86 +1,94 @@
-    * Open a valid command prompt:
-        * **Basic:** run the `x64 Native Tools Command Prompt for VS2017` shortcut bundled with VS 2017, or
-        * **Advanced:** run `vcvarsall.bat x64` from a blank `cmd` session
+**Note:** If you get the error: `'atomic.h' file not found`, try installing `libx32stdc++-12-dev`.
 
-- MacOS
-    * Install the latest XCode (from the App Store or the [Xcode website](https://developer.apple.com/xcode/))
-    * Install XCode command-line tools `xcode-select --install`
-    * Install [Homebrew](https://brew.sh/) 
-    * Install LLVM through Homebrew with: `brew install llvm@14`
-    * Make sure the LLVM binaries and the linker are added to your `$PATH` environmental variable (see `brew info llvm@14`)
+**Note:** If you get the error: `'sys/wait.h' file not found` and `'stdio.h' file not found`, try installing `g++`.
 
-- GNU/Linux and other \*Nix
-    * For Linux: clang and llvm (version 11.1, 12, 13, or 14; using your distro's package manager)
-    * Note: If an atomic.h error occurs, also add `libx32stdc++-12-dev`
-    * For FreeBSD: `pkg install bash git llvm14`
-    * Make sure the LLVM binaries and the linker are added to your `$PATH` environmental variable
+**Note:** If you get the error: `'linux/futex.h' file not found`, try installing `linux-headers`.
 
-## Building Odin
-Now, it's time to build Odin and get started!
+**Note:** On some distributions, you need to add a "-dev" suffix to the llvm and clang packages like so: `llvm14-dev`, `clang14-dev` to get the development files.
 
-### For Windows
-There's a couple prerequisites here. First, make sure you have Visual Studio installed; you have to compile Odin from source, and Odin also requires `link.exe` from VS anyway. The necessary LLVM components for Windows are included in the Odin repository.
+**Note:** The package names may differ slightly depending on your distribution.
 
-Now, it's time to build Odin and get started! Open the X64 Visual Studio command prompt ([if you don't typically use it, here's how to find it](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)) and navigate to the directory where you downloaded Odin. Run the `build.bat` file, and you should have a successfully built Odin compiler!
+### BSD
+- On FreeBSD, run: `pkg install bash git llvm14`
+- [Add the LLVM binaries to your `$PATH`.](#add-llvm-to-your-path)
+- Navigate to the Odin directory inside your terminal.
+- Run `make` to build Odin. That creates the `odin` binary.
+- [Add the Odin directory to your `$PATH`.](#add-odin-to-your-path)
+- Use `odin --help` to learn how to run your first Odin program.
 
-To use Odin `link.exe` is required to be in the PATH of the callee as mentioned, this can either be achieved but calling Odin from the X64 Visual Studio command prompt or by calling the vcvarsall.bat (with x64 as an argument) script either in your shell or in your build script.
+## Add LLVM to your PATH
```

Comment:
This is only required on non-Windows systems.

Reply:
I removed that section and instead made it a step for all platforms that require it.